tests/test_models.py

```diff
@@ -2655,6 +2655,21 @@ def test_set_privacy(self):
         assert_false(self.project.is_public)
         assert_equal(self.project.logs[-1].action, NodeLog.MADE_PRIVATE)
 
+    @mock.patch('website.project.model.mails.queue_mail')
+    def test_set_privacy_sends_mail(self, mock_queue):
+        self.project.set_privacy('private', self.consolidate_auth)
```

Comment:
This really should be two tests, as the second assert is not really testing anything, will separate them. The idea is that one assert is checking whether default behavior (```skip_mail``` is set to ```False``` on default), and the second assert checks whether setting the flag as false still sends the email (as a counterpart to ```test_set_privacy_skips_mail```)